index.html

```diff
@@ -32,6 +34,6 @@ <h1>Guessing Game</h1>
       </div>
       <button id="reset">Reset</button>
     </main>
-    <script src="index.js"></script>
+    <script src="/intro-to-programming-section-5/index.js"></script>
```

Comment:
You should revert this line back to its original state. Because index.js is in the same directory as index.html, you don't need to add the "/intro-to-programming-section-5/" into the file href.